source/declarations.tex

```diff
@@ -1683,8 +1683,8 @@
 body.
 \begin{example}
 \begin{codeblock}
-auto  f() { } // OK, return type is \tcode{void}
-auto* g() { } // error, cannot deduce \tcode{auto*} from \tcode{void()}
+auto  f() { } // OK: return type is \tcode{void}
```

Comment:
This one should stay as a comma. (We're not saying it's OK because the return type is `void`, we're saying both (a) it's OK, and (b) the return type is `void`.)